corpus/accounts/models.py

```diff
@@ -138,6 +142,8 @@ class ExecutiveMember(models.Model):
     github = models.CharField(
         max_length=39, blank=True, null=True, verbose_name="GitHub Username"
     )
+    hide_github = models.BooleanField(default=False, null=True, blank=True)
+    hide_linkedin = models.BooleanField(default=False, null=True, blank=True)
```

Comment:
For booleans, don't allow null and blank values. The default should be False, and let them opt in to true.